tableauserverclient/server/endpoint/flow_runs_endpoint.py

```diff
@@ -5,20 +5,26 @@
 
 import logging
 
+from typing import List, Optional, Tuple, TYPE_CHECKING, Union
+
 logger = logging.getLogger("tableau.endpoint.flowruns")
 
+if TYPE_CHECKING:
+    from ..server import Server
+    from ..request_options import RequestOptions
 
 class FlowRuns(QuerysetEndpoint):
-    def __init__(self, parent_srv):
+    def __init__(self, parent_srv: "Server") -> None:
         super(FlowRuns, self).__init__(parent_srv)
+        return None
 
     @property
-    def baseurl(self):
+    def baseurl(self) -> str:
         return "{0}/sites/{1}/flows/runs".format(self.parent_srv.baseurl, self.parent_srv.site_id)
 
     # Get all flows
     @api(version="3.10")
-    def get(self, req_options=None):
+    def get(self, req_options: Optional["RequestOptions"]=None) -> Tuple[List[FlowRunItem], PaginationItem]:
```

Comment:
We use this format throughout our endpoints, I was thinking we might want to refactor this into a type with a typevar like `TSResponse[T] = Tuple[List[T], PaginationItem] -> TSResponse[FlowRunItem]` but I haven't dug into TypeVars deeply. No need to do it for this PR

Reply:
I will play around with that in a new PR, we can do it after all of these are in.